test/src/org/exist/TestUtils.java

```diff
@@ -17,23 +16,42 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.StandardCopyOption;
 import java.util.Iterator;
 import java.util.Optional;
 import java.util.stream.Stream;
+
 import org.exist.util.ConfigurationHelper;
 
 /**
- * Created by IntelliJ IDEA.
- * User: wolf
- * Date: Oct 29, 2007
- * Time: 4:28:26 PM
- * To change this template use File | Settings | File Templates.
+ * Utility functions for working with tests
  */
 public class TestUtils {
-	
-    public static void cleanupDB() {
 
+    /**
+     * Default Admin username used in tests
+     */
+    public static final String ADMIN_DB_USER = "admin";
```

Comment:
I could reference them, however they have different purposes; so changing the ones in the SecurityManager in future could cause unexpected changes in the tests; which is why I did not do that.